src/send_data.py

```diff
@@ -0,0 +1,51 @@
+from ast import alias
+from venv import create
+from xmlrpc.client import boolean
+from sensor_temp import getTemp
+from sensor_motion import getMotion 
+from alarm import alarm_on
+from button import button_pressed
+import requests
+import json
+import time
+import datetime
+
+URL = 'http://127.0.0.1:8000'
+DEVICE_ID = '1'
+
+## SEND DATA OUT   
+def create_sensor_data_dict():
+    sensor_temp = getTemp()
+
```

Comment:
`sensor_temp = getTemp()`
`getTemp()` will raise an error. You must try catch this too.

Reply:
Done